<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerTableWidgetBase</class>
 <widget class="QWidget" name="QgsComposerTableWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QToolBox" name="mToolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>237</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Table</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="mLayerLabel">
           <property name="text">
            <string>Layer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mLayerComboBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="mComposerMapLabel">
           <property name="text">
            <string>Composer map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mComposerMapComboBox"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="mMaxNumFeaturesLabel">
           <property name="text">
            <string>Maximum columns</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mMaximumColumnsSpinBox"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="mMarginLabel">
           <property name="text">
            <string>Margin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mMarginSpinBox"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="mShowGridCheckBox">
         <property name="text">
          <string>Show grid</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="mGridStrokeWidthLabel">
           <property name="text">
            <string>Grid stroke width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mGridStrokeWidthSpinBox"/>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="mGridColorLabel">
         <property name="text">
          <string>Grid color</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QgsColorButton" name="mGridColorButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="mHeaderFontPushButton">
         <property name="text">
          <string>Header Font...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="mContentFontPushButton">
         <property name="text">
          <string>Content Font...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
